src/test/java/org/tikv/txn/KVClientTest.java

```diff
@@ -0,0 +1,39 @@
+package org.tikv.txn;
+
+import static org.junit.Assert.fail;
+
+import com.google.protobuf.ByteString;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.tikv.common.TiConfiguration;
+import org.tikv.common.TiSession;
+
+public class KVClientTest {
+  private TiSession session;
+
+  @Before
+  public void setUp() {
+    TiConfiguration conf = TiConfiguration.createDefault();
+    conf.setTest(true);
+    try {
+      session = TiSession.create(conf);
+    } catch (Exception e) {
+      fail("TiKV cluster may not be present");
+    }
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    if (session != null) {
+      session.close();
+    }
+  }
+
+  @Test
```

Comment:
```suggestion
  @Ignore
```

Reply:
Current CI only support run RawKV test, cause with TTL enabled TxnKV API is not supported.

```
[2021-09-22T02:51:20.675Z] [2021/09/22 10:41:19.973 +08:00] [WARN] [session.go:1579] ["run statement failed"] [schemaVersion=0] [error="tikv aborts txn: Error(Other(\"[src/storage/mod.rs:952]: can't sched txn cmd(prewrite) with TTL enabled\"))"] [session="{\n  \"currDBName\": \"\",\n  \"id\": 0,\n  \"status\": 2,\n  \"strictMode\": true,\n  \"user\": null\n}"]

[2021-09-22T02:51:20.675Z] goroutine 1 [running]:

[2021-09-22T02:51:20.675Z] runtime/debug.Stack(0x0, 0xc000f9f758, 0x0)

[2021-09-22T02:51:20.675Z] 	/usr/local/go/src/runtime/debug/stack.go:24 +0x9f

[2021-09-22T02:51:20.675Z] runtime/debug.PrintStack()

[2021-09-22T02:51:20.675Z] 	/usr/local/go/src/runtime/debug/stack.go:16 +0x25

[2021-09-22T02:51:20.675Z] github.com/pingcap/tidb/session.mustExecute(0x4243ed8, 0xc0000ff400, 0x3cdfa6a, 0x22, 0x0, 0x0, 0x0)

[2021-09-22T02:51:20.675Z] 	/home/jenkins/agent/workspace/build_tidb_multi_branch_master/go/src/github.com/pingcap/tidb/session/bootstrap.go:1742 +0xd9

[2021-09-22T02:51:20.675Z] github.com/pingcap/tidb/session.doDDLWorks(0x4243ed8, 0xc0000ff400)

[2021-09-22T02:51:20.675Z] 	/home/jenkins/agent/workspace/build_tidb_multi_branch_master/go/src/github.com/pingcap/tidb/session/bootstrap.go:1604 +0x5f

[2021-09-22T02:51:20.675Z] github.com/pingcap/tidb/session.bootstrap(0x4243ed8, 0xc0000ff400)

[2021-09-22T02:51:20.675Z] 	/home/jenkins/agent/workspace/build_tidb_multi_branch_master/go/src/github.com/pingcap/tidb/session/bootstrap.go:373 +0x353

[2021-09-22T02:51:20.675Z] github.com/pingcap/tidb/session.runInBootstrapSession(0x4206650, 0xc00041a540, 0x3da3d28)

[2021-09-22T02:51:20.675Z] 	/home/jenkins/agent/workspace/build_tidb_multi_branch_master/go/src/github.com/pingcap/tidb/session/session.go:2579 +0x297

[2021-09-22T02:51:20.675Z] github.com/pingcap/tidb/session.BootstrapSession(0x4206650, 0xc00041a540, 0x0, 0x0, 0x0)

[2021-09-22T02:51:20.675Z] 	/home/jenkins/agent/workspace/build_tidb_multi_branch_master/go/src/github.com/pingcap/tidb/session/session.go:2432 +0xd1b

[2021-09-22T02:51:20.675Z] main.createStoreAndDomain(0x5fb89c0, 0x3d0289c, 0x2c)

[2021-09-22T02:51:20.675Z] 	/home/jenkins/agent/workspace/build_tidb_multi_branch_master/go/src/github.com/pingcap/tidb/tidb-server/main.go:290 +0x189

[2021-09-22T02:51:20.675Z] main.main()

[2021-09-22T02:51:20.675Z] 	/home/jenkins/agent/workspace/build_tidb_multi_branch_master/go/src/github.com/pingcap/tidb/tidb-server/main.go:191 +0x26a

[2021-09-22T02:51:20.675Z] [2021/09/22 10:41:19.973 +08:00] [FATAL] [bootstrap.go:1743] ["mustExecute error"] [error="tikv aborts txn: Error(Other(\"[src/storage/mod.rs:952]: can't sched txn cmd(prewrite) with TTL enabled\"))"] 
```